openmetadata-ui/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -420,3 +420,5 @@ export const MAX_VISIBLE_OWNERS_FOR_FEED_TAB = 4;
 export const MAX_VISIBLE_OWNERS_FOR_FEED_CARD = 2;
 
 export const BREADCRUMB_SEPARATOR = '/';
+
+export const DEFAULT_READ_TIMEOUT = 12;
```

Comment:
We are planning to remove this file. Always keep things in specific files, please move this to another constants file, alert-specific constant file if it's present.